docs/content/docs/deployment/azure-frs.md

```diff
@@ -50,7 +50,7 @@ const schema = {
     },
     dynamicObjectTypes: [ /*...*/ ],
 }
-const frsClient = new FrsClient(config);
+const frsClient = new AzureClient(config);
```

Comment:
should these variable names be changed too?

Reply:
Good catch! Updated, also updating the references in testing.md rn